deepchem/utils/batch_utils.py

```diff
@@ -0,0 +1,90 @@
+"""
+Utility Functions for computing features on batch.
+"""
+import numpy as np
+
+
+def coulomb_matrix_features(X_b: np.ndarray,
+                            distance_max: float = -1,
+                            distance_min: float = 18,
+                            n_distance: int = 100):
+    """Computes the values for different Feature Layers on given batch.
```

Comment:
Can you expand this explanation? What does the batching do? What models is it needed for? What is the scientific justification and do we have references?

Reply:
Done